roles/wireguard/templates/iface_config.conf.j2

```diff
@@ -17,28 +17,42 @@ Table = {{ item.table }}
 
 {% for peer in item.peers %}
 [Peer]
-PublicKey = {{ hostvars[peer].wg_pubkeys[item.iface_name] }}
+PublicKey = {{ hostvars[peer].wg_pubkeys[item.iface_name] | default(wireguard_config_directory[peer][item.iface_name].wg_pubkey) }}
 {% if item.psk_enabled is defined %}
-PreSharedKey = {{ hostvars[peer].pre_shared_keys[item.iface_name] }}
+PreSharedKey = {{ hostvars[peer].pre_shared_keys[item.iface_name] | default(wireguard_config_directory[peer][item.iface_name].pre_sharing_key) }}
 {% endif %}
-AllowedIPs = {{ (hostvars[peer].wg_ifaces
-    | selectattr('iface_name', 'equalto', item.iface_name)
-    | map(attribute='ip_addresses_cidr')
-    | first
-    + (item.allowed_ips | default([])))
-    | list
-    | join(', ')
-}}
-Endpoint = {{
-    [ (hostvars[peer].wg_ifaces
-      | selectattr('iface_name', 'equalto', item.iface_name)
-      | map(attribute='peering_ip')
-      | first),
+AllowedIPs = {{
+    (
       (hostvars[peer].wg_ifaces
       | selectattr('iface_name', 'equalto', item.iface_name)
-      | map(attribute='peering_port')
-      | first)
-    ] | join(':')
+      | map(attribute='ip_addresses_cidr')
+      | first
+      + (item.allowed_ips | default([])))
```

Comment:
Please indent either the `|` or `+` lines. Currently, the precedence is not obvious.

Reply:
Fixed locally.